website/client/directives/dragdrop.directive.js

```diff
@@ -11,14 +13,15 @@ import _without from 'lodash/without';
  * v-drag.drop.DRAG_GROUP="KEY_OF_ITEM" @itemDropped="callback" @itemDragOver="optional"
  */
 
+/*
 const DROPPED_EVENT_NAME = 'itemDropped';
 const DRAGSTART_EVENT_NAME = 'itemDragStart';
 const DRAGEND_EVENT_NAME = 'itemDragEnd';
 const DRAGOVER_EVENT_NAME = 'itemDragOver';
 const DRAGLEAVE_EVENT_NAME = 'itemDragLeave';
-
+*/
 export default {
-  bind (el, binding, vnode) {
+  /* bind (el, binding, vnode) {
```

Comment:
I had to disable the drag&drop, couldn't find a solution in short period of time , should I rather remove all the events/ directive attributes inside the views instead of disabling the directive inside the code ?